lib/src/rules.dart

```diff
@@ -46,6 +46,7 @@ import 'package:linter/src/rules/prefer_const_constructors.dart';
 import 'package:linter/src/rules/prefer_contains.dart';
 import 'package:linter/src/rules/prefer_final_fields.dart';
 import 'package:linter/src/rules/prefer_final_locals.dart';
+import 'package:linter/src/rules/prefer_interpolation_to_compose_strings.dart';
```

Comment:
It appears that you've mixed in changes from an earlier PR. Please separate them.